.github/workflows/integration_tests.yaml

```diff
@@ -7,6 +7,17 @@ jobs:
     integration_tests:
         runs-on: ubuntu-latest
 
+        services:
+            postgres:
+                image: postgres:11.7
+                env:
+                    POSTGRES_PASSWORD: root
+                    POSTGRES_DB: pehapkari_cz
+                ports:
+                    - 5432:5432
```

Comment:
Interesting... it looks exactly like Docker configuration

Reply:
It is very close to it. Source: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices